bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
Before you do that, I would suggest filing an issue on ESLint. Maybe that's something we could add to the public API? I'm not 100% sure how useful that would be to others, but it's a possibility.